appserver/ejb/ejb-http-remoting/client/pom.xml

```diff
@@ -86,5 +86,13 @@
             <groupId>javax.json</groupId>
             <artifactId>javax.json-api</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.glassfish.jersey.inject</groupId>
+            <artifactId>jersey-hk2</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.glassfish.jersey.media</groupId>
+            <artifactId>jersey-media-json-jackson</artifactId>
```

Comment:
Since the client should depend on JSONB I believe the correct dependency (and probably a runtime one) is `jersey-media-json-binding`

Reply:
That worked, thanks for pointing it out.